README.md

```diff
@@ -1,110 +1,46 @@
-
-To turn on generic Didact completion notifications (off by default), access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Disable Notifications` checkbox. If unchecked, additional notifications will occur when Didact actions are activated in the IDE.
-
-To have Didact open a window with the default Didact file on startup, access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Open Default Tutorial At Startup` checkbox.
-
-![Didact Settings](./images/settings.jpg)
-
-## Auto-complete for Didact in Markdown files
-
-To help you set up Didact tutorials, we've added a number of auto-completions (accessible by pressing `Ctrl+Space` while editing a Markdown file) to provide most of the basic things you'll need.
-
-![Didact Auto-Completion Demo](./images/didact-uri-completion-demo.gif)
-
-Included completions:
-
-* Insert Validate All button
-* Insert Requirements label
-* Insert link to install required VS Code extension
-* Insert link to create temporary folder as WS root
-* Insert link to start didact from File in Extension folder
-* Start a new didact link (inserts `didact://?` and then offers additional completions
-
-From a `didact://?` start, you also get:
-
-* Start terminal with name
-* Send named terminal some text
-* Send named terminal a Ctrl+C
-* Close terminal with name
-* Non-didact command
-* Check CLI for some returned text
-* Check CLI for some success (no text)
-* Check for required extension
-* Check for root folder in the WS
-* Scaffold project
-* Start Didact from Currently Selected File
-
-## A Note about Auto-complete for Didact in AsciiDoc files
-
-This functionality requires an AsciiDoc extension to be installed in your Workspace. We have seem intermittent issues with these extensions, including this [popular AsciiDoc extension.](vscode:extension/joaompinto.asciidoctor-vscode). We have noticed that the Didact extension does not always start when this extension is installed.
-
-When it works, we should have access to the following completions.
-
-Outside a didact link, you get:
-
-* Start a new didact link (inserts `didact://?` and then offers additional completions
-* Insert Link to Start Didact from File in Extension folder
-* Insert Link to Install Required VS Code extension
-
-From a `didact://?` start, you get:
-
-* Start terminal with name
-* Send named terminal some text
-* Send named terminal a Ctrl+C
-* Close terminal with name
-* Non-didact command
-* Check CLI for some returned text
-* Check CLI for some success (no text)
-* Check for required extension
-* Check for root folder in the WS
-* Scaffold project
-* Start Didact from Currently Selected File
-* Validate All Didact Requirements
-* Add Temporary Folder as WS Root
```

Comment:
Can you point where in the wiki there is the information about auto-completion for AsciiDoc?

Reply:
See previous response

Reply:
Moved the docs to https://github.com/redhat-developer/vscode-didact/wiki/Didact-Auto-complete and will update when we release a new version that has the newer autocomplete functionality